### PR TITLE
Capture the new nextGame and use it rather than calling nextGame again

### PR DESCRIPTION
**Pull request by Tilps**

### CODE REVIEW

**Comment by Tilps** [comment]
Untested, just compiles.  Could potentially be using invalid pointers for all I know :P

**Comment by Tilps** [comment]
Tested, no issues found.  Appears to solve the problem of dropped match games.

File: lc0_main.go [hunk 1]
```diff
@@ -548,6 +557,9 @@ func nextGame(httpClient *http.Client, count int) error {
 			errCount := 0
 			for {
 				time.Sleep(60 * time.Second)
+				if nextGame.Type == "Done" {
```
Comment (Tilps) [inline lc0_main.go]: This seems like a prudent measure, but seemed doubly so after this change.

File: lc0_main.go [hunk 2]
```diff
@@ -557,6 +569,9 @@ func nextGame(httpClient *http.Client, count int) error {
 					}
 				}
 				if err != nil || ng.Type != nextGame.Type || ng.Sha != nextGame.Sha {
```
Comment (borg323) [inline lc0_main.go]: Its probably clearer to just do `if err!= nil` here (keeping the old code) and later test for the other conditions to do `pendingNextGame = &ng`

**Pull request merged**